docs/tutorials/fairplay.md

```diff
@@ -76,3 +76,24 @@ player.getNetworkingEngine().registerResponseFilter((type, response) => {
   response.data = shaka.util.Uint8ArrayUtils.fromBase64(responseText).buffer;
 });
 ```
+
+## DRM Engine
+
+Some FairPlay support needs to be detected in order to change source to HLS if FairPlay is supported (AFAIK currently FairPlay is supported only within HLS and not publicly supported in DASH). You can query EME yourself, directly, and only query FairPlay, rather than the full list of key systems probed by `DrmEngine`. 
```

Comment:
Split lines at 80 characters.  Also, there's no need to mention `DrmEngine`; you could say `probeSupport` instead.